### PR TITLE
feature/strategist_profile_and_issuer_profile_pages_added

### PR DESCRIPTION
**Pull request by hemendra-sakpal-algobulls**

### CODE REVIEW

File: docs/member/issuer-profile.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Issuer Profile
+
+## 1. What is an Issuer?
+
+A legal entity that manages, registers, and sells securities to support its operations financially is an Issuer. There can be many types of Issuers - Corporations, Domestic or Foreign Governments and Investment trusts. Issuers develop securities such as Bonds, Debentures, G-Secs & NCDS. Issuers are legally responsible for reporting financial conditions, material developments, and other operational activities required by the jurisdiction's regulations.
```
Comment (akhil-jain-algobulls) [inline docs/member/issuer-profile.md:5]: `NCDS`

check if the last letter S is a part of the NCD acronym or it is a plural form.

if it is a plural form then S should be in small caps, like NCDs
if it is a part of the acronym, then this is correct

File: docs/member/issuer-profile.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+The AlgoBulls Issuer Profile gives you a detailed view of the issuer’s name, bonds available, transactions, average yield and more. Check the Issuer profile here.
```
Comment (akhil-jain-algobulls) [inline docs/member/issuer-profile.md:7]: `Check the Issuer profile here.`

`here` was supposed to be a link here right?

File: docs/member/issuer-profile.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+### iii. Bio
+
+A detailed summary of the Issuer.
+
+[![Issuer Profile](imgs/issuer_profile_summary.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/issuer_profile_summary.png)
+
+### iv. Number of Bonds
+
+View the number of Bonds available from the Issuer
```
Comment (akhil-jain-algobulls) [inline docs/member/issuer-profile.md:28]: `View the number of Bonds available from the Issuer`

this line does not have a full stop (.) in the end. but other points above ad below have them

what is the consistency that we have followed for other pages? we keep the full stop or not? please change as per that

File: docs/member/issuer-profile.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+A legal entity that manages, registers, and sells securities to support its operations financially is an Issuer. There can be many types of Issuers - Corporations, Domestic or Foreign Governments and Investment trusts. Issuers develop securities such as Bonds, Debentures, G-Secs & NCDS. Issuers are legally responsible for reporting financial conditions, material developments, and other operational activities required by the jurisdiction's regulations.
+
+The AlgoBulls Issuer Profile gives you a detailed view of the issuer’s name, bonds available, transactions, average yield and more. Check the Issuer profile here.
+
+
+## 2. Issuer Profile Summary
+
+### i. Name
+
+This section displays the Issuer Name & Logo.
```
Comment (akhil-jain-algobulls) [inline docs/member/issuer-profile.md:14]: somewhere we have used `&`

somewhere we have used `and`

please follow consistency and change everywhere

File: docs/member/issuer-profile.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+## 3. The Issuer Bond Cards
+
+The Bond card includes the Issuer’s name, Security Name, Yield Per Year, Security & Taxation criteria. You can also view the price, rating, maturity date, Credit Outlook, IP Frequency and Allotment Date. You may also Buy or Sell the bond from the Bonds card. Click on the card to view more details.
+
+[![Issuer Profile](imgs/issuer_profile_issuer_card.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/issuer_profile_issuer_card.png)
+
+### i. Pagination
+
+You can view the bond cards in various page view modes, such as 4 Page/8 Page/ 12 Page. You can also view the Bond cards in the Regular Mode or List Mode.
```
Comment (akhil-jain-algobulls) [inline docs/member/issuer-profile.md:76]: `4 Page/8 Page/ 12 Page`

spacing issues

File: docs/member/strategist-profile.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
+# Strategist Profile
+
+## Introduction
+AlgoBulls provides a variety of trading strategies, but the brains behind these strategies are market experts who work extremely hard to create complex yet simple strategies. The Strategist profile will help you recognize the strategist's work and strategies. Let’s take a look at what the strategist profile offers
```
Comment (akhil-jain-algobulls) [inline docs/member/strategist-profile.md:4]: `create complex yet simple strategies`

@pushpakdagade i understand what we are trying to convey here, but there is something wrong with the usage of these terms like this. please help to change this

Comment (akhil-jain-algobulls) [reply docs/member/strategist-profile.md:4]: `Let’s take a look at what the strategist profile offers`

there should have been a full stop (.) or colon (:) here at the end

File: docs/member/strategist-profile.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+## Strategist Profile Summary
+
+Get the strategist’s profile summary details such as the total number of strategies, deployments, trades, wins, total loss, long trades, short trades, max gain, and more.
+
+[![Strategist Profile](imgs/strategiest_profile_summary.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/strategiest_profile_summary.png)
+
+
+
+## Explore the market experts’ strategies
```
Comment (akhil-jain-algobulls) [inline docs/member/strategist-profile.md:20]: Are the number of lines left blank above this correct ?

File: docs/member/strategist-profile.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+Explore the strategies developed by the strategist as per the marketplace. Analyze the growth, trading volume, and total trades of the strategists' strategy using the Heatmap. The marketplace is divided into 3 parts - Retail, Premium & HNI.
```
Comment (akhil-jain-algobulls) [inline docs/member/strategist-profile.md:22]: strategist
strategist's
strategists'

these imply 3 different meanings. Please get this checked across the whole page.

Ideally we should not be using strategist and strategists' in the same point.

File: docs/member/strategist-profile.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+[![Strategist Profile](imgs/strategiest_profile_heat_map.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/strategiest_profile_heat_map.png)
+
+The Retail Marketplace displays all retail strategies. View the Premium Marketplace to see all premium strategies, and the HNI Marketplace features all HNI Strategies.
```
Comment (akhil-jain-algobulls) [inline docs/member/strategist-profile.md:26]: `View the Premium Marketplace to see all premium strategies, and the HNI Marketplace features all HNI Strategies.`

this should be
`The Premium Marketplace features all premium strategies, and the HNI Marketplace features all HNI Strategies.`
or
`View the Premium Marketplace to see all premium strategies, and the HNI Marketplace to see all HNI Strategies.`

We should not mix both

File: docs/member/strategist-profile.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+## Strategy Cards
+
+View the Strategy Cards created by the market expert here.
```
Comment (akhil-jain-algobulls) [inline docs/member/strategist-profile.md:30]: `market expert`

do we mean a strategist? if yes, then we should mention strategist here

File: docs/member/strategist-profile.md (hunk 9 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md:26]: Changed to this line:
View the Premium Marketplace to see all premium strategies, and the HNI Marketplace to see all HNI Strategies.
that is line 3 in the suggestion

File: docs/member/strategist-profile.md (hunk 8 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md:22]: strategists' changed to strategist's

File: docs/member/strategist-profile.md (hunk 7 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md:20]: Extra lines removed

File: docs/member/strategist-profile.md (hunk 6 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md:4]: Full stop added

File: docs/member/issuer-profile.md (hunk 5 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/issuer-profile.md:76]: Done

File: docs/member/issuer-profile.md (hunk 4 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/issuer-profile.md:14]: Done

File: docs/member/issuer-profile.md (hunk 3 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/issuer-profile.md:28]: Full stop added

File: docs/member/issuer-profile.md (hunk 2 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/issuer-profile.md:7]: Link added

File: docs/member/issuer-profile.md (hunk 1 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/issuer-profile.md:5]: It is plural, changes made

File: docs/member/strategist-profile.md (hunk 10 above)
Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md]: Done

File: docs/member/strategist-profile.md (hunk 6 above)
Comment (pushpakdagade) [reply docs/member/strategist-profile.md]: @akhil-jain-algobulls yes the language doesn't seem proper. Inform the same to Mayuri and get her to revise the language.

Comment (akhil-jain-algobulls) [reply docs/member/strategist-profile.md]: `create complex yet simple strategies` @hemendra-sakpal-algobulls please get this part revised

Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md]: @akhil-jain-algobulls, the updated content is pushed, please check

File: docs/member/strategist-profile.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
+# Strategist Profile
+
+## Introduction
+AlgoBulls provides a variety of trading strategies, but the brains behind these strategies are strategist who work extremely hard to create complex yet simple strategies. The Strategist profile will help you recognize the strategist's work and strategies. Let’s take a look at what the strategist profile offers.
```
[the remark this answers is not in the document]
Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md]: The word strategist was replaced with market experts as per one of the previous suggestions

[the remark this answers is not in the document]
Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md]: The sentence changed to `but the brains behind these strategies are strategists who work extremely hard to create complex yet simple strategies`

[the remark this answers is not in the document]
Comment (hemendra-sakpal-algobulls) [reply docs/member/strategist-profile.md]: @akhil-jain-algobulls

**Review by pushpakdagade — APPROVED** [review]

**Review by pushpakdagade — APPROVED** [review]

**Pull request merged**